src/SurveyService.Validation/Group/CreateGroupRequestValidator.cs

```diff
@@ -0,0 +1,40 @@
+using FluentValidation;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.SurveyService.Validation.Group.Interfaces;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Group;
+
+public class CreateGroupRequestValidator : AbstractValidator<CreateGroupRequest>, ICreateGroupRequestValidator
+{
+  public CreateGroupRequestValidator(
```

Comment:
please, add check:
if Deadline is null, HasRealTimeResult is true

Reply:
Shouldn't we add the same for CreateSingleQuestionRequestValidator?

Reply:
should. i forgot((

Reply:
Done